packages/smart-vaults/src/utils/MultiSignerAuth.sol

```diff
@@ -111,13 +120,13 @@ abstract contract MultiSigner {
      * @param signer_ The owner raw bytes to register.
      * @param index_ The index to register the signer.
      */
-    function addSigner(Signer calldata signer_, uint8 index_) public onlyAuthorized {
-        MultiSignerStorage storage $ = _getMultiSignerStorage();
+    function addSigner(Signer calldata signer_, uint8 index_) external onlyAuthorized {
+        MultiSignerAuthStorage storage $ = _getMultiSignerAuthStorage();
```

Comment:
so the extra contract layer is for 7201 right?

does it make more sense to just save the storage slot directly to signers everywhere? since that's all we ever use anyway
```
        MultiSigner storage $ = _getMultiSignerAuthStorage().signers;
```